components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -648,17 +649,22 @@ void CosmeticFiltersJSHandler::OnHiddenClassIdSelectors(
   TRACE_EVENT1("brave.adblock", "OnHiddenClassIdSelectors", "url", url_.spec());
 
   base::Value::List* hide_selectors = result.FindList("hide_selectors");
-  DCHECK(hide_selectors);
+  if (!hide_selectors) {
+    return;
+  }
 
   base::Value::List* force_hide_selectors =
       result.FindList("force_hide_selectors");
-  DCHECK(force_hide_selectors);
+  if (!force_hide_selectors) {
+    return;
+  }
 
   if (force_hide_selectors->size() != 0) {
```

Comment:
```suggestion
  if (force_hide_selectors && force_hide_selectors->size() != 0) {
```

Reply:
There is two options how to process invalid fields: to skip or to continue.
If we go for 2 let's use this logic everywhere.

Reply:
+1 to this instead of the early return, especially since there are other parts of this function that can still work without `force_hide_selectors`